docs/extend/extend-apiml/api-mediation-infinispan.md

```diff
@@ -0,0 +1,55 @@
+# Using Infinispan as a storage solution through the Caching service
+
+As an API developer, you can configure Infinispan as a storage solution through the Caching service. This article describes how to configure your storage solution for Infinispan.
+You can configure Infinispan for high availability as well as to replicate data to provide data durability and availability.
+
+- [Understanding Infinispan](#understanding-infinispan)
+- [Infinispan configuration](#infinispan-configuration)
+
+## Understanding Infinispan
+
+Infinispan is a storage solution that stores data structures in key-value pairs. The API Caching service uses hash sets, where each
+service storing data via the Caching service has its own hash, and each data entry is a key-value entry within that service's Infinispan hash set.
+
+For more information on Infinispan, see the [official Infinispan documentation](https://infinispan.org/documentation/).
+
+### Infinispan replica instances
+
+Infinispan can be used with one standalone instance. For data durability, however, a Cross-Site Replication configuration is recommended.
+Cross-site replication guarantees service continuity in the event of outages or disasters and presents client applications with a single point of access to data in globally distributed caches.
+
+For more information on Infinispan replication and how to configure a replica instance, see the [official Infinispan Cross-site Replication documentation](https://infinispan.org/docs/stable/titles/xsite/xsite.html).
```

Comment:
When I started working on the doc I misunderstood what was for. It's actually for data recovery. I will remove it since it's not even configurable. Thanks for catching it